client/extensions/woocommerce/app/promotions/promotion-models.js

```diff
@@ -159,20 +159,27 @@ const productSaleModel = {
 		labelText: translate( 'Product & Sale Price' ),
 		cssClass: 'promotions__promotion-form-card-primary',
 		fields: {
-			salePrice: {
-				component: CurrencyField,
-				labelText: translate( 'Product Sale Price' ),
-				isRequired: true,
-			},
-		}
+			salePrice: (
+				<CurrencyField
+					labelText={ translate( 'Product Sale Price' ) }
+					isRequired={ true }
+				/>
+			),
+		},
 	},
 	conditions: {
 		labelText: translate( 'Conditions', { context: 'noun' } ),
 		cssClass: 'promotions__promotion-form-card-conditions',
 		fields: {
-			startDate,
-			endDate,
-		}
+			startDate: (
+				<DateField
+					labelText={ translate( 'Start Date' ) }
+					isEnableable={ true }
+					disabledDays={ [ { before: new Date() } ] }
```

Comment:
neat! 🌟 , and `isEnableable` could stand on its own without `={ true }`

Reply:
Replaced all instances of `isEnableable`, thanks